shoop/core/management/commands/shoop_makemessages.py

```diff
@@ -0,0 +1,44 @@
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2015, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+"""
+Extract translatable strings from shoop
+"""
+from __future__ import unicode_literals
+
+import logging
+import subprocess
+
+import os
+from django.conf import settings
+from django.core import management
+from django.core.management import BaseCommand
+
+
+logging.basicConfig(level=logging.INFO)
+
+
+class Command(BaseCommand):
```

Comment:
I still feel this shouldn't be a management command, for the reasons I outlined earlier (which got no reply):

> I don't feel a management command is the right fit for this job; management commands are supposed to be run in the context of a finished project, and i18n extraction for the Shoop codebase shouldn't require a project (be it `shoop_workbench` or anything else).